.gitignore

```diff
@@ -67,6 +67,10 @@ log.txt
 simWorkspace/
 tmp/
 null/
+
+#mac
+*.DS_Store
```

Comment:
I suggest to make that part of the global gitignore file. See various hints on Google or just here https://stackoverflow.com/questions/7335420/global-git-ignore

Reply:
I have no opinion about that part. What do you think @Dolu1990 ?

Reply:
I have configured the global gitignore file 👍